canu/generate/switch/config/config.py

```diff
@@ -107,6 +108,7 @@
     "templates",
 )
 env = Environment(
+    autoescape=select_autoescape(),
```

Comment:
What's the reasoning behind using select_autoescape() vs setting True?

https://jinja.palletsprojects.com/en/3.0.x/api/?highlight=select_autoescape#jinja2.select_autoescape

says that it's based on template extension. Ours is always `.j2`which would be `True`.  Looks like this would only affect `.html`, `.xml` and `.txt`.  Shouldn't we just set this to `True`?

Reply:
One of the templates failed to render just using `True`.. I didn't dig in, but this seemed to produce the same effect without failing.